.github/workflows/release.yaml

```diff
@@ -0,0 +1,150 @@
+name: Release build
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  # If you change this value, please change it in the following files as well:
+  # /Dockerfile
+  # /dev.Dockerfile
+  # /make/builder.Dockerfile
+  # /taprpc/Dockerfile
+  # /tools/Dockerfile
+  # /.github/workflows/*.yml
+  GO_VERSION: 1.21.0
+
+jobs:
+  main:
+    name: Release build
+    runs-on: ubuntu-latest
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: setup go ${{ env.GO_VERSION }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: '${{ env.GO_VERSION }}'
+
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+
+      - name: build release for all architectures
+        run: SKIP_VERSION_CHECK=1 make release tag=${{ env.RELEASE_VERSION }}
+
+      - name: Create Release
+        uses: lightninglabs/gh-actions/action-gh-release@2021.01.25.00
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ env.RELEASE_VERSION }}
+          name: tapd ${{ env.RELEASE_VERSION }}
+          draft: true
+          prerelease: false
+          files: tapd-${{ env.RELEASE_VERSION }}/*
+          body: |
+            # Database Migrations
+            TODO
+
+            # Verifying the Release
+
+            In order to verify the release, you'll need to have `gpg` or `gpg2` installed on your system. Once you've obtained a copy (and hopefully verified that as well), you'll first need to import the keys that have signed this release if you haven't done so already: 
+
+            ```
+            curl https://raw.githubusercontent.com/lightninglabs/taproot-assets/master/scripts/keys/roasbeef.asc | gpg --import
```

Comment:
This endpoint doesn't exist atm.

Reply:
Are you ok with me copying over your `roasbeef.asc` from lnd?

Reply:
@Roasbeef I've added your public key in this PR's latest commits. This will ensure that a hyperlink in our release verification example is valid. Please let me know if that's ok with you.